features/api.feature

```diff
@@ -1487,4 +1487,253 @@ Feature: Browsing via API
     And the JSON response should be:
     """
     {"error":"metadata parameter not found"}
-    """
\ No newline at end of file
+    """
+
+  #  @api_add_item
```

Comment:
You probably don't need this test if you limit the routes. It's best to avoid putting commented code in the repo wherever possible because 99% of the time it never gets uncommented. If you create something that you *think* you might want to keep in the future, either create a new branch and commit it there, or stash it.